echo-telemetry/echo-telemetry.gradle

```diff
@@ -22,6 +22,7 @@ dependencies {
   implementation project(':echo-core')
   implementation "com.netflix.spinnaker.kork:kork-jedis"
   implementation 'com.google.protobuf:protobuf-java-util'
+  implementation "com.netflix.spinnaker.kork:kork-exceptions"
```

Comment:
seems unrelated?

Reply:
This is where com.netflix.spinnaker.security.AuthenticatedRequest lives...The code doesn't compile without it.  https://github.com/spinnaker/kork/blob/master/kork-security/src/main/java/com/netflix/spinnaker/security/AuthenticatedRequest.java

Reply:
or maybe...I'm getting this wrong and have copy-pasted from other similar changes...gimme a minute.

Reply:
Sorry, it does in fact compile with nothing additional here.  I think it's correct to add
```
implementation "com.netflix.spinnaker.kork:kork-security"
```
instead of kork-exceptions though, since kork-security is actually where AuthenticatedRequest lives....like this: https://github.com/spinnaker/echo/pull/1091/commits/8021a567c85de1ea82c6d82ffa8b4a8c10c953d3

Reply:
thanks, looking good now 👍